### ISSUE TITLE
GHDL mingw64-llvm35 fails on AppVeyor

### ISSUE THREAD

**Issue by Paebbels** [ghdl/ghdl#702]
**Description**
The build for platform MinGW64 + LLVM 3.5 fails on AppVeyor.

**Expected behavior**
Do not fail :).

**Context**
![image](https://user-images.githubusercontent.com/956109/48917277-3b235600-ee86-11e8-82c7-cc0f880f818c.png)

- OS: Windows Server 2016
- Origin: Commit SHA: many latest commits

**Additional context**
MinGW64 has updated LLVM to another version, so LLVM 3.5 is not available anymore.

**Comment by Paebbels** [comment]
According to my local MinGW64 installation, the default LLVM version is 7.0.

    Patrick Lehmann@PLC2-Paebbels MINGW64 ~
    $ llvm-config.exe --version
    7.0.0

@1138-4EB  
Have I missed something? When and why did LLVM jump to 7.0?

@tgingold  
Does GHDL already support LLVM 7.0?

**Comment by Paebbels** [comment]
LLVM 7.0 support is proposed in #703.

**Comment by 1138-4EB** [comment]
@Paebbels, IDK. They jumped from 5 to 7 in less than a year. However, Fedora 29 (which was released in October) is using LLVM 6 yet.

BTW, it is almost a year since the last GHDL release was published. @tgingold, @Paebbels, should we consider releasing v0.36 in December?

**Comment by Paebbels** [comment]
@tgingold is working on the unconstrained record stuff to get it finished, so we can announce, *full* support for UVVM and OSVVM AXI models.

**Comment by tgingold** [comment]
As far as I can see, there are still failures on Appveyor.  Not a build failure, but some tests fail.

**Comment by 1138-4EB** [comment]
@Paebbels do you have those test failures on your machine? I'm trying to tell if we have a problem with appveyor only, or with LLVM on Windows.

**Comment by Paebbels** [comment]
Yes, I need to run the test environment on my machine. As far as I remember, this was not so easy ...
